cassandane/testrunner.pl

```diff
@@ -249,13 +246,15 @@ sub usage
     }
     elsif ($a eq '-f')
     {
-        $format = shift;
-        usage unless defined $runners{$format};
+        my $format = shift;
+        usage unless defined $formatters{$format};
+        $want_formats{$format} = 1;
     }
     elsif ($a =~ m/^-f(\w+)$/)
     {
-        $format = $1;
-        usage unless defined $runners{$format};
+        my $format = $1;
+        usage unless defined $formatters{$format};
+        $want_formats{$format} = 1;
```

Comment:
I would love to overhaul this code... but your changes are clearly improvements.  They're just building on a mess. :)